webrtc.html

```diff
@@ -6610,105 +6610,84 @@ <h2>Methods</h2>
         </section>
       </div>
       <p>The <dfn>RTCRtpContributingSource</dfn> and
-      <dfn>RTCRtpSynchronizationSource</dfn> objects contain information about
-      a given contributing source (CSRC) or synchronization source (SSRC),
-      including the most recent time a
+      <dfn>RTCRtpSynchronizationSource</dfn> dictionaries contain information
+      about a given contributing source (CSRC) or synchronization source (SSRC)
+      respectively, including the most recent time a
       packet that the source contributed to was played out. The browser MUST
       keep information from RTP packets received in the previous 10 seconds.
       When the first audio frame contained in an RTP packet is delivered to the
       <code><a>RTCRtpReceiver</a></code>'s <code><a>MediaStreamTrack</a></code>
       for playout, the user agent MUST queue a task to update the relevant
-      <code><a>RTCRtpContributingSource</a></code> and
-      <code><a>RTCRtpSynchronizationSource</a></code> objects based on the
-      contents of the packet. The
-      <code><a>RTCRtpSynchronizationSource</a></code> object corresponding to
-      the SSRC identifier is updated each time, and if the RTP packet contains
-      CSRC identifiers, then the <code><a>RTCRtpContributingSource</a></code>
-      objects corresponding to those CSRC identifiers are also updated.</p>
+      information for the <code><a>RTCRtpContributingSource</a></code> and
+      <code><a>RTCRtpSynchronizationSource</a></code> dictionaries based on the
+      contents of the packet. The information relevant to the
+      <code><a>RTCRtpSynchronizationSource</a></code> dictionary corresponding
+      to the SSRC identifier, is updated each time, and if the RTP packet
+      contains CSRC identifiers, then the information relevant to the
+      <code><a>RTCRtpContributingSource</a></code> dictionaries corresponding to
+      those CSRC identifiers is also updated.</p>
```

Comment:
This still talks about updating dictionaries upon receiving RTP packets. But since we don't want a "live" state anymore, what is updated is some internal set of dictionaries, correct? The ones returned by getContributingSources() being copies not affected by this update.

We still want the "queue a task to update the dictionaries" type of language to indicate that if an RTP packet is received midst-JS task execution cycle we don't see this until the next cycle?
getContributingSources() is always value-equal to getContributingSources() in a single cycle?

Just checking to make sure Chromium implementation still requires a cache.

Reply:
This has already been answered - the answer is yes.